docs/porting/connectivity/MeshMAC.md

```diff
@@ -183,6 +206,161 @@ Member|Description
 `parent_id` | Service user ID used to indentify the MAC service user. Optional.
 `phyMTU` | Maximum Transmission Unit (MTU) used by MAC. Standard 802.15.4 MAC must set 127.
 
+## SW MAC Information element's and Enhanced ACK support
+
+This chapter introduce how to use extented features.
+### Initialize API
```

Comment:
All that content should already be in the Doxygen fields in the header.
Therefore you should drop ALL function descriptions from here, and only talk about the porting process, pitfalls, and link to the Doxygen for API reference.

Reply:
@SeppoTakalo Removing those it will make this file quite a short and less information.

Reply:
Short does not mean there is less information.
You should already have all this in the header file. If not, fix your header files first.

Doxygen should be good enough, so that this is not needed.